saleor/graphql/schema.graphql

```diff
@@ -1831,11 +1831,75 @@ type Image {
   alt: String
 }
 
+type Invoice implements Node & ObjectWithMetadata {
+  id: ID!
+  created: DateTime!
+  number: String!
+  order: Order!
+  url: String
+  privateMetadata: [MetadataItem]!
+  metadata: [MetadataItem]!
+  privateMeta: [MetaStore]! @deprecated(reason: "DEPRECATED: Will be removed in Saleor 2.11. use the `privetaMetadata` field instead. ")
```

Comment:
There is no point in adding deprecated fields; if possible we shouldn't do that, but I'm not sure if it will be possible in Graphene. We have `ObjectWithMetadata` that adds them. @tomaszszymanski129 When implementing this I'd check if we can somehow manually exclude these fields.

Reply:
If we want to implement `ObjectWithMetadata` we should add these fields, even if it's pointless 🤷‍♂️

Reply:
i will have a look, maybe we can exclude it. I agree there is no point in making use of deprecated field.